src/test/scala/chiselTests/BundleLiteralSpec.scala

```diff
@@ -22,14 +23,39 @@ class BundleLiteralSpec extends ChiselFlatSpec with Utils {
     val c = MyEnum()
   }
 
+  class LongBundle extends Bundle {
+    val a = UInt(48.W)
+    val b = SInt(32.W)
+    val c = FixedPoint(16.W, 4.BP)
+  }
+
+  "bundle literals" should "pack" in {
+    assertTesterPasses{ new BasicTester {
+      val bundleLit = (new MyBundle).Lit(_.a -> 42.U, _.b -> false.B, _.c -> MyEnum.sB)
+      bundleLit.litOption should equal (Some(169))  // packed as 42 (8-bit), false=0 (1-bit), sB=1 (1-bit)
+      chisel3.assert(bundleLit.asUInt() === bundleLit.litOption.get.U)  // sanity-check consistency with runtime
+
+      val longBundleLit = (new LongBundle).Lit(
+        _.a -> 0xDEADDEADBEEFL.U, _.b -> -0x0BEEF00DL.S(32.W), _.c -> 4.5.F(16.W, 4.BP))
```

Comment:
maybe bracket for `(-0x0BEEF00DL).S`? I don't know does `-` have priority or `.S` does, but Intellij blames for this.